osf/utils/tokens.py

```diff
@@ -0,0 +1,105 @@
+from rest_framework import status as http_status
```

Comment:
It looks like this file was removed in this commit (https://github.com/CenterForOpenScience/osf.io/commit/d4f5586a719e37c5c2b3a107008e3e986f98678b) because it duplicates code. I'm guessing it shouldn't be re-added and is a byproduct of branching/merging?

Reply:
It looks like it is being used in that file though. Ln40, Ln80

Reply:
Sorry, I'm not talking about the import. I was trying to say that `osf.utils.tokens.py` was removed in https://github.com/CenterForOpenScience/osf.io/commit/d4f5586a719e37c5c2b3a107008e3e986f98678b because it duplicates code.

Reply:
good catch